### PR TITLE
audit: log to /var/log/kube-apiserver/audit.log on the host

### PR DESCRIPTION
**Pull request by sttts**
The directory is a host mount, created with 0755.

### CODE REVIEW

File: bindata/v3.11.0/kube-apiserver/pod.yaml [hunk 1]
```diff
@@ -48,3 +50,6 @@ spec:
   - hostPath:
       path: /etc/kubernetes/static-pod-resources/kube-apiserver-pod-REVISION
     name: resource-dir
+  - hostPath:
+      path: /var/log/kube-apiserver
+    name: audit-dir
```
Comment (deads2k) [inline bindata/v3.11.0/kube-apiserver/pod.yaml]: nit, I like trailing newlines

Comment (sttts) [reply bindata/v3.11.0/kube-apiserver/pod.yaml]: fixed

File: bindata/v3.11.0/kube-apiserver/pod.yaml [hunk 2]
```diff
@@ -14,9 +14,12 @@ spec:
     image: ${IMAGE}
     imagePullPolicy: IfNotPresent
     terminationMessagePolicy: FallbackToLogsOnError
-    command: ["hypershift", "openshift-kube-apiserver"]
+    command: ["/bin/bash", "-xec"]
```
Comment (sttts) [inline bindata/v3.11.0/kube-apiserver/pod.yaml]: @deads2k added the `-e` to be sure that the chmod succeeded. Please retag.

**Pull request merged**